kotlinx-coroutines-core/jvm/src/Dispatchers.kt

```diff
@@ -41,14 +41,16 @@ public actual object Dispatchers {
      *
      * Depending on platform and classpath it can be mapped to different dispatchers:
      * - On JS and Native it is equivalent of [Default] dispatcher.
-     * - On JVM it either Android main thread dispatcher, JavaFx or Swing EDT dispatcher. It is chosen by
+     * - On JVM it is either Android main thread dispatcher, JavaFx or Swing EDT dispatcher. It is chosen by
      *   [`ServiceLoader`](https://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html).
      *
-     * In order to work with `Main` dispatcher, following artifact should be added to project runtime dependencies:
+     * In order to work with `Main` dispatcher, the following artifacts should be added to project runtime dependencies:
      *  - `kotlinx-coroutines-android` for Android Main thread dispatcher
      *  - `kotlinx-coroutines-javafx` for JavaFx Application thread dispatcher
      *  - `kotlinx-coroutines-swing` for Swing EDT dispatcher
      *
+     * In order to test with `Main` dispatcher, add the `kotlinx-coroutines-test` artifact to project test dependencies.
```

Comment:
It does clearly explain what "testing with main dispatcher" is.  Consider replacing "In order to test" with "In order to be able to set a custom main dispatcher for test purpose" of something like that.